src/patternfly/components/Menu/menu.scss

```diff
@@ -202,33 +202,29 @@
     margin-bottom: var(--pf-c-menu--c-divider--MarginBottom);
   }
 
-  &.pf-m-flyout {
-    overflow: visible;
-
-    .pf-c-menu {
-      position: absolute;
-      top: var(--pf-c-menu--m-flyout__menu--Top);
-      right: auto;
-      left: var(--pf-c-menu--m-flyout__menu--Left);
-
-      &.pf-m-top {
-        top: auto;
-        bottom: var(--pf-c-menu--m-flyout__menu--m-top--Bottom);
-      }
+  // nested flyout menu
+  .pf-c-menu.pf-m-flyout,
+  &.pf-m-flyout .pf-c-menu {
+    position: absolute;
+    top: var(--pf-c-menu--m-flyout__menu--Top);
+    right: var(--pf-c-menu--m-flyout__menu--Right);
+    bottom: var(--pf-c-menu--m-flyout__menu--Bottom);
+    left: var(--pf-c-menu--m-flyout__menu--Left);
 
-      &.pf-m-left {
-        right: var(--pf-c-menu--m-flyout__menu--m-left--Right);
-        left: auto;
-      }
-    }
 
     .pf-c-menu__content {
       overflow-y: visible;
     }
+  }
 
-    .pf-c-menu__list-item {
-      position: relative;
-    }
+  &.pf-m-top {
+    --pf-c-menu--m-flyout__menu--Top: auto;
+    --pf-c-menu--m-flyout__menu--Bottom: var(--pf-c-menu--m-flyout__menu--m-top--Bottom);
+  }
+
+  &.pf-m-left {
+    --pf-c-menu--m-flyout__menu--Right: var(--pf-c-menu--m-flyout__menu--m-left--Right);
+    --pf-c-menu--m-flyout__menu--Left: auto;
```

Comment:
We also have `.pf-m-top` and `.pf-m-left` defined twice in the nav css. Are we able to just define the modifier classes here and use vars in the nav css if we need to change something?